source/geometries/Next100SiPM.cc

```diff
@@ -71,6 +76,34 @@ void Next100SiPM::Construct()
 
   GeometryBase::SetLogicalVolume(sipm_logic_vol);
 
+  // COATING /////////////////////////////////////////////
+
+  G4LogicalVolume* coating_logic_vol;
+  if (with_coating_) {
+    G4String coating_name = sipm_name + "_WLS";
+
+
+    G4Box* coating_solid_vol =
+      new G4Box(coating_name, sipm_width/2., sipm_length/2., coating_thickn_/2.);
+
+    G4Material* coating_mt_ = MaterialsList::TPB();
```

Comment:
In nexus, the convention for local variables is not to use trailing underscore. We just use it for global variables.

Reply:
Methinks you mean "the convention is to use trailing underscores for **member** variables":

``` c++
int this_is_a_global_variable = 0; // There's one for the whole program

void some_function() {
    int this_is_a_local_variable = 0; // It's only in scope inside this function
}

class some_class {
    int this_is_a_member_variable = 0; // Every instance of this class gets a separate one
};
```